source/technical/networks/networks_basic.rst

```diff
@@ -68,49 +70,58 @@ attached. You have a few options of how to mitigate risk of compromise.
    experiment, feel free to file a `Help Desk
    <https://www.chameleoncloud.org/user/help/>`_ ticket.
 
-Limit bound interfaces
-^^^^^^^^^^^^^^^^^^^^^^
-
-Instead of binding a web service to all interfaces (e.g. ``0.0.0.0`` for IPv4,
-``::`` for IPv6), consider listening only on the node's private IP, which is not
-routable from the public Internet. If you can, listening on localhost
-(``127.0.0.1``) is even safer. Most web services have a way to specify the bind
-address and some default to binding on all interfaces, which is often insecure.
-
 Firewall
 ^^^^^^^^
 
-Ubuntu ships with a simple firewall utility called ``ufw``. When setting up your
-environment, it is easy to add a few rules to ensure that your node only allows
-inbound connections on a few ports. The following example allows only SSH from
-the public Internet, but any internal traffic on private subnets is still
-permitted.
+Chameleon supported Ubuntu and CentOS images are preconfigured with a firewall
```

Comment:
```suggestion
Chameleon-supported Ubuntu and CentOS images are preconfigured with a firewall
```

In general, when such noun phrases are used as adjectives, a hyphen is appropriate. https://www.grammarly.com/blog/hyphen/ - see section on nouns+past participles